tests/lean/run/bv_decide_rewriter.lean

```diff
@@ -25,7 +25,6 @@ def mem_subset (a1 a2 b1 b2 : BitVec 64) : Bool :=
 theorem mem_subset_refl : mem_subset a1 a2 a1 a2 := by
   unfold mem_subset
   bv_normalize
-  sorry
```

Comment:
Note that this test previously normalized to `(a2 + (~~~a1 + 1#64)).ult 0#64 = true` in one of the branches, so now with `not_lt_zero` this sorry is no longer necessary.